src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Create.cshtml

```diff
@@ -64,8 +64,9 @@
             @:<div class="form-group">
                 @:<div class="checkbox">
                     @:<label>
-                        @:<input asp-for="@(Model.ModelTypeName).@property.PropertyName" /> @@Html.DisplayNameFor(model => model.@GetValueExpression(property)))
-                    @:</label>
+    @:<input asp-for="@(Model.ModelTypeName).@property.PropertyName" /> @@Html.DisplayNameFor(model => model.@(Model.ModelTypeName).@GetValueExpression(property)))
+    @:
+</label>
```

Comment:
@barld The newline here is causing the tests to fail. 
It should be `@:</label>`  instead of 
```
@:
</label>
```